apps/ledger-live-mobile/src/families/elrond/components/Details/index.ts

```diff
@@ -0,0 +1,6 @@
+import OperationDetailsExtra from "./Details";
+
+import { getURLFeesInfo } from "./helpers/getURLFeesInfo";
+import { getURLWhatIsThis } from "./helpers/getURLWhatIsThis";
+
+export default { getURLWhatIsThis, getURLFeesInfo, OperationDetailsExtra };
```

Comment:
getURLWhatIsThis, getURLFeesInfo

I can't find them use anywhere on code ? Did I miss something ?

Reply:
Used here it seems
apps/ledger-live-mobile/src/screens/OperationDetails/index.tsx
apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx